packages/frontend/src/pages/public/CreateHousePage.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button, Card, Text, Modal, Input, Backdrop } from '@nextui-org/react';
```

Comment:
This file, alongside ManageAccountPage, should be placed in a new `private` directory as they are only for users logged in.